lightwood/data/timeseries_transform.py

```diff
@@ -30,16 +31,25 @@ def transform_timeseries(
     :param timeseries_settings: A `TimeseriesSettings` object.
     :param target: The name of the target column to forecast.
     :param mode: Either "train" or "predict", depending on what phase is calling this procedure.
+    :param use_dask: If `True` and Dask is installed, lightwood will use Dask to compute the transformations.
+    :param mp_row_cutoff: if dataset size is bigger than this amount, parallel compute procedures will be used.
```

Comment:
Probably not worth having `mp_row_cutoff` as a parameter, seems to specific. Tbh I'd consider just canning it and doing mp no matter the number of rows to reduce complexity instead of reverting back to hardcoding 500.